cypress/e2e/tests/migration/dynamic-report/dependencies/mta_2008_validation.test.ts

```diff
@@ -0,0 +1,80 @@
+import { Analysis } from "../../../../models/migration/applicationinventory/analysis";
+import {
+    clickWithinByText,
+    deleteByList,
+    getRandomAnalysisData,
+    getRandomApplicationData,
+    login,
+    selectItemsPerPage,
+} from "../../../../../utils/utils";
+import { rightSideMenu } from "../../../../views/analysis.view";
+import { AnalysisStatuses, button } from "../../../../types/constants";
+import { AppDependency } from "../../../../types/types";
+import { Dependencies } from "../../../../models/migration/dynamic-report/dependencies/dependencies";
+
+describe(["@tier3"], "Testing dependencies bugs", function () {
+    let applicationsList: Array<Analysis> = [];
+
+    before("Test title", function () {
+        login();
+
+        cy.fixture("application").then((appData) => {
+            cy.fixture("analysis").then((analysisData) => {
+                const bookServerApp = new Analysis(
+                    getRandomApplicationData("mta_2008_book_server_", {
+                        sourceData: appData["bookserver-app"],
+                    }),
+                    getRandomAnalysisData(analysisData["analysis_for_openSourceLibraries"])
+                );
+                applicationsList.push(bookServerApp);
+            });
+        });
+        cy.fixture("application").then((appData) => {
+            cy.fixture("analysis").then((analysisData) => {
+                const dayTraderApp = new Analysis(
+                    getRandomApplicationData("mta_2008_daytrader_", {
+                        sourceData: appData["daytrader-app"],
+                    }),
+                    getRandomAnalysisData(analysisData["source+dep_analysis_on_daytrader-app"])
+                );
+                applicationsList.push(dayTraderApp);
+
+                applicationsList.forEach((application) => application.create());
+            });
+        });
+    });
+
+    beforeEach("Load data", function () {
+        cy.fixture("application").then(function (appData) {
+            this.appData = appData;
+        });
+        cy.fixture("analysis").then(function (analysisData) {
+            this.analysisData = analysisData;
+        });
+    });
+
+    it("Testing bug MTA-2008", function () {
```

Comment:
Could you include a short description of bug MTA-2088 for better readability ?

Reply:
Added link to the bug, hope it is enough